### PR TITLE
Fixed Avatar Design Issue in Editor in Twenty Nineteen theme

### PR DESCRIPTION
**Pull request by shail-mehta**
After Applying Patch Border radius issue will solved.

**Trac ticket:** [59285](https://core.trac.wordpress.org/ticket/59285)

**Screenshots:**

Before Patch: https://prnt.sc/AXe3wMAR7cr9
After Patch : https://prnt.sc/N74p7U-mj6Io

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-content/themes/twentynineteen/style-editor.css [hunk 1]
```diff
@@ -1605,3 +1605,8 @@ ul.wp-block-archives li ul,
   padding-right: 0;
   width: 100%;
 }
+
+/* Post Author Block  */
+.editor-styles-wrapper .avatar, .editor-styles-wrapper .wp-block-post-author__avatar img {
+  border-radius: 50%;
+}
```
Comment (mukeshpanchal27) [inline src/wp-content/themes/twentynineteen/style-editor.css]: The CSS will auto generate based on SCSS when it build, Could you please remove this changes?

File: src/wp-content/themes/twentynineteen/style-editor.scss [hunk 2]
```diff
@@ -1020,3 +1020,13 @@ $group-block-background__padding: $font__size_base;
 		width: 100%;
 	}
 }
+
+/* Post Author Block  */
```
Comment (mukeshpanchal27) [inline src/wp-content/themes/twentynineteen/style-editor.scss]:
```suggestion
/** === Post Author Block === */
```

Use same formate for comment.

File: src/wp-content/themes/twentynineteen/style-editor.scss [hunk 3]
```diff
@@ -1020,3 +1020,13 @@ $group-block-background__padding: $font__size_base;
...
+.editor-styles-wrapper {
+	
+	& .avatar,
+	& .wp-block-post-author__avatar img {
+		border-radius: 50%;
+	}
+
+}
```
Comment (mukeshpanchal27) [inline src/wp-content/themes/twentynineteen/style-editor.scss]: Add empty line at the end of CSS.

**Review by mukeshpanchal27 — CHANGES_REQUESTED** [review]
Thanks @shail-mehta for the PR, left some feedback that needs to address.

**Comment by shail-mehta** [comment]
Thank you for the feedback @mukeshpanchal27.

I have Updated PR as per mentioned Feedback changes.
- Reverted style-editor.css Changes
- Used same format for comment.
- Added Empty line at the end of css

File: src/wp-content/themes/twentynineteen/style-editor.css [hunk 4]
```diff
@@ -1604,4 +1604,4 @@ ul.wp-block-archives li ul,
   padding-left: 0;
   padding-right: 0;
   width: 100%;
-}
+}
```
Comment (sabernhardt) [inline src/wp-content/themes/twentynineteen/style-editor.css]: Please restore the empty line at the end of the file.

(Whoever commits the change could compile this stylesheet from the SCSS, but the file should remain unchanged if it is not compiled within the pull request.)

File: src/wp-content/themes/twentynineteen/style-editor.scss [hunk 5]
```diff
@@ -1020,3 +1020,15 @@ $group-block-background__padding: $font__size_base;
...
+/** === Post Author Block === */
+
+.editor-styles-wrapper {
+	
+	& .avatar,
+	& .wp-block-post-author__avatar img {
+		border-radius: 50%;
+	}
+
+}
```
Comment (sabernhardt) [inline src/wp-content/themes/twentynineteen/style-editor.scss]:
```suggestion
.avatar,
.wp-block-post-author__avatar img {
	border-radius: 100%;
}
```

Twenty Nineteen's editor styles automatically add the `.editor-styles-wrapper` selector. Also, while 50% may look like 100%, Twenty Nineteen uses 100% for the front end.

**Review by sabernhardt — CHANGES_REQUESTED** [review]

File: src/wp-content/themes/twentynineteen/style-editor.scss (hunk 5 above)
Comment (shail-mehta) [reply src/wp-content/themes/twentynineteen/style-editor.scss]: @sabernhardt I have updated changes as per mentioned.

File: src/wp-content/themes/twentynineteen/style-editor.css (hunk 4 above)
Comment (shail-mehta) [reply src/wp-content/themes/twentynineteen/style-editor.css]: I have Restored Last empty line.

**Comment by carolinan** [comment]
When I apply the patch on WordPress 6.5-alpha-56966 I am getting:
```
5426.diff:16: new blank line at EOF.
+
warning: 1 line adds whitespace errors.
```

And there are two blank lines at the end of the style-editor.scss file.

Other than that, the PR solves the problem with the avatar radius.

**Pull request closed without merging**